### PR TITLE
TSE feedback

### PR DESCRIPTION
**Pull request by BenciR**
# Smashing Magazine Layout

### Here we implemented:
- Latest Articles
- Latest Post
- Ads
- Person of the week
- Guides 
- Footer

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Invalid :red_square: 

## Hi @EraaniR & @Gnwin :wave: 

- [ ] You should make sure that your repo looks professional before review

- There is no custom README added
- Your Pull Requests does not have a meaningful title and short summary

Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).

**This Code Review Request will be marked as invalid in your Dashboard, so please make your repo professional and resubmit for review once you are ready!**

**I'm @taofeekolalere on slack**
Happy Coding :man_technologist:

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: :wave: Hello Team!! The heat-map project is looking really neat! 

I love the little details with the effects you put on some of the elements! :+1: 

Things to notice:
:100: Descriptive README
:100: Descriptive Pull Request
:100: Descriptive Commit messages.
:100: Nice touch with the animations
:100: Great heat map grayscale
:100: Correctly using a local version of the linters
:100: 2 space indentation!!! thanks for this! it improves readability!
:100: Using a reset.css file!

## Requested Changes :recycle: 

### [Mandatory] :red_circle: 

- Project title mismatch.
image

- [x] MIT Licence, and Issue links are broken.

- [x] Use semantic HTML tags for the main section instead of a div container. (look inline comment)

- [x] Use semantic HTML tags for the articles instead of a div container. (look inline comment)

### [OPTIONAL] :large_blue_circle: 

- [x] This section is not acting as expected... **But it could be an issue of screen sizes. If it is please omit this issue.**
image

- [x] Use meaningful class/selector names. (look inline comment)

### [COMMENTS]
- [x] For future projects improve branch names. Example:
**Actual**: ```latest-post```.
**Improved** :+1: : ```smashing-magazine-clone```.

- [x] Improve Commit messages. This will improve the history of the project. Will give the reader a better understanding of the decisions made. And help the programmer remember what he was trying to accomplish.
**Actual**: ```finishing des section```
**Improved** :+1: : ```Finish navbar section```

## Keep up the great work Team :muscle: and Happy Coding :smiley: :computer:!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+        <li class="navbar-list_link"><a href="#" class="n-d">Membership</a></li>
+        <li class="navbar-list_link"><a href="#" class="n-d">Jobs</a></li>
+        <li class="navbar-list_link more active"><a href="#" class="n-d">More</a><img
+            src="https://www.smashingmagazine.com/images/icons/menu-cat.svg" class="cat" alt="cat"></li>
+      </ul>
+      <input type="search" class="searchbar" placeholder="Search articles..." />
+    </nav>
+  </header>
+
+  <div class="latest-articles d-flex c-fff ff-Jadi">
```
Comment (jubaan) [inline index.html]: - [x] Wrap the main content in ```main``` semantic HTML tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+            class="profileimg" alt="person profile pic" /> Jonathan Kelley<span class="light">wrote</span></div>
+        <p class="text">Desiging with reduced motion for motion sensitives<span class="com">Leave a comment</span></p>
+        <p class="hashtags">
+          <span class="ht"># API<sup class="sup"></sup></span>
+          <span class="ht"># CMS<sup class="sup"></sup></span>
+          <span class="ht"><sup class="sup"></sup></span>
+        </p>
+      </div>
+    </div>
+    <div class="article a-left">
```
Comment (jubaan) [inline index.html:78]: - [x] Use ```article``` tag wherever is needed there are various lines of code that's need it so look around. Having a class article is perfect. The recommendation here is that if there is an HTML tag for certain types of content use it over a generic ```div``` tag or other generic HTML tags.

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## :+1: Great work team!! I see you have worked on the required changes!!

You may now merge your feature branch with master!

## Awesome work!! :clap: :clap: :clap: 

### [COMMENTS]

- [x] The MIT information should be related to you. Also update the date. This is because you're the creators and giving the authorization under MIT.

image

**Pull request merged**